servicecontrol/upgrades/5to6.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Upgrade ServiceControl from Version 5 to Version 6
+summary: Instructions on how to upgrade ServiceControl from version 5 to 6
+reviewed: 2023-11-30
+isUpgradeGuide: true
+component: ServiceControl
+---
+
+ServiceControl version 6 updates the underlying data storage from RavenDB 5 to RavenDB 6. Once upgraded, the database cannot be downgraded.
+
+Users hosting ServiceControl in containerized infrastructure must also make changes to ensure data is accessible in the new version.
+
+## Prerequisites
+
+Before upgrading to ServiceControl version 6, ensure all ServiceControl instances have first been upgraded to version 5.11.0.
+
+## Container changes
+
+When using the [`servicecontrol-ravendb` container image](https://hub.docker.com/r/particular/servicecontrol-ravendb), the following changes must be made when upgrading to ServiceControl version 6:
+
+1. The [required volume mount location](/servicecontrol/ravendb/containers.md#required-settings) must be changed:
+    * Version 5: `/opt/RavenDB/Server/RavenData`
+    * Version 6: `/var/lib/ravendb/data`
+    * If the container mount path is not changed, the container will refuse to start.
+2. On Linux hosts, the permissions for the data volume must be changed to user id 999.
+    * With access to the volume, this can be accomplished using the following command:
+        ```bash
+        chown -R 999:999 /path/to/data/volume
+        ```
+    * This is due to RavenDB version 6 running as a non-root user.
+    * Users running containers via Docker Desktop do not need to do this as Docker Desktop abstracts away group/owner permissions.
```

Comment:
Do we need to mention Docker Desktop in an upgrade guide? Isn't this documentation aimed at production?

```suggestion
```